src/Computation.hs

```diff
@@ -224,6 +227,56 @@ regionX' :: CompFn a
 regionX' [] [LN' lineLeft, LN' lineRight] = TNum $ regionX lineLeft lineRight
 regionX' v o = error' "regionX" v o
 
+-- returns the middle of the square as a starting position
+midSquare' :: CompFn a
+midSquare' [] [S' b] = TPt((xs' b),(ys' b))
+
+rightSquare' :: CompFn a
+rightSquare' [] [S' b] = TPt((xs' b) + (side' b)/2,(ys' b) + (side' b)/2)
+
+leftSquare' :: CompFn a
+leftSquare' [] [S' b] = TPt((xs' b) - (side' b)/2,(ys' b) + (side' b)/2)
+
+toRightSquare' :: CompFn a
+toRightSquare' [] [S' b] = TPt(((xs' b) + 200),(ys' b))
+
+toLeftSquare' :: CompFn a
+toLeftSquare' [] [S' b] = TPt(((xs' b) - 200),(ys' b))
+
+toAboveSquare' :: CompFn a
+toAboveSquare' [] [S' b] = TPt((xs' b),((ys' b)+125))
+
+toBelowSquare' :: CompFn a
+toBelowSquare' [] [S' b] = TPt((xs' b),((ys' b)-125))
+
+startArrow' :: CompFn a
+startArrow' [] [A' a] = TPt((startx' a),(starty' a))
+
+endArrow' :: CompFn a
+endArrow' [] [A' a] = TPt((endx' a),(endy' a))
+
+reverseEndArrow' :: CompFn a
+reverseEndArrow' [] [A' a] = let sx = (startx' a)
+                                 sy  = (starty' a)
+                                 ex = (endx' a)
+                                 ey = (endy' a)
+                              in TPt(sx - (ex - sx), sy - (ey - sy))
```

Comment:
rename function to something generic like `negV`
can simplify last expr to just `(-ex, -ey)`